### PR TITLE
[FR] 8.12 Release Preparation update Main Branch to 8.13

### PR DESCRIPTION
**Pull request by eric-forte-elastic**
## Issues
* https://github.com/elastic/ia-trade-team/issues/229

## Summary

Also fixes typo in building the integrations (integrations.py function `build_integrations_schemas`) that would cause the ml packages to not necessarily be updated.

- [x] update packages.yml
- [x] update integrations manifest
- [x] update integrations schema
- [x] update lock-versions.yml
- [x] update workflows
- [x] lock api_schemas
- [x] update beats and ecs schemas
- [x] add migrate function
- [x] update stack-schema-map

_merged after 8.12 branch creation_
_unit testing will fail at first because the 8.12 branch does not exist yet and is created prior to merge_

### CODE REVIEW

**Review by terrancedejesus — APPROVED** [review]
Approved.

File: .github/workflows/lock-versions.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ on:
           description: 'List of branches to lock versions (ordered, comma separated)'
           required: true
           # 7.17 was intentionally skipped because it was added late and was bug fix only
-          default: '8.3,8.4,8.5,8.6,8.7,8.8,8.9,8.10,8.11'
+          default: '8.3,8.4,8.5,8.6,8.7,8.8,8.9,8.10,8.11,8.12'
```
Comment (Mikaayenson) [inline .github/workflows/lock-versions.yml:9]: @terrancedejesus @eric-forte-elastic when are we going to start dropping versions?

**Review by w0rk3r — APPROVED** [review]

**Review by Mikaayenson — APPROVED** [review]
LGTM, but we need to start considering dropping or define when we will.

File: .github/workflows/lock-versions.yml (hunk 1 above)
Comment (terrancedejesus) [reply .github/workflows/lock-versions.yml:9]: We discussed this during the training/exercise. Previously it was undecided when we were comfortable with dropping support to our agreed current-3 scope. Maybe we create a discussion and if all agrees, drop a certain list of minors.

**Pull request merged**